ql/src/java/org/apache/hadoop/hive/ql/HookRunner.java

```diff
@@ -56,6 +57,7 @@ public class HookRunner {
   HookRunner(HiveConf conf, SessionState.LogHelper console) {
     this.conf = conf;
     this.hooks = new HiveHooks(conf, console);
+    addNecessaryHooks();
```

Comment:
just add it directly

Reply:
`addNecessaryHooks()` will have all the hooks that are absolutely required from now on. Thought this would be a good way to go about. But let me know if I have to change and directly add it.

Reply:
Done